include/fmt/format.h

```diff
@@ -2673,6 +2673,18 @@ template <typename Char>
 void vformat_to(buffer<Char>& buf, basic_string_view<Char> fmt,
                 basic_format_args<buffer_context<type_identity_t<Char>>> args,
                 locale_ref loc) {
+  using detail::arg_formatter;
+  using detail::buffer_appender;
+  using detail::custom_formatter;
+  using detail::default_arg_formatter;
+  using detail::get_arg;
+  using detail::locale_ref;
+  using detail::parse_format_specs;
+  using detail::specs_checker;
+  using detail::specs_handler;
+  using detail::to_unsigned;
+  using detail::type;
+  using detail::write;
```

Comment:
I'm still not sure, if this is really needed by the standard or if this is just an implementation deficit. At least it looks ugly. 

From reading "C++ Templates - The Definitive Guide" and the standard, it is clear that the local class is a _templated entity_. I need further studies of the standard to find an answer to the question: which names within this so-called 'temploid' are bound at which time.

I'd rather prefer to make this 'temploid' a proper class template.

Reply:
Which local class are you referring to?

Reply:
`struct format_handler` within `vformat_to` in format.h, currently line 2649.

The problem is explored with this stripped-down code example here [https://godbolt.org/z/34Ph9dee9](https://godbolt.org/z/34Ph9dee9). Fortunately I got this compiled with gcc (modules-branch) for the first time just a few minutes ago, so that I have a second opinion on the matter now: gcc doesn't require this 'symbolic-linking' of names into the body of `vformat_to` and all of my studies of the standard seem to corroborate that. Therefore I consider this an implementation bug in latest msvc. This symbolic linking is the only workaround that works at all. Qualified name lookup doesn't. And only local classes within function templates are affected.

Reply:
This is acknowledged as bug in msvc by realiable sources familiar with the compiler: name-binding is happening in phase 2 instead of phase 1 in this particular case. In other words: too late to find anything in namespace `detail`.

Reply:
Could you add a short comment clarifying that this is a workaround for an msvc bug?